enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -153,6 +154,10 @@ const (
 	SamplerSleepDuration  = 1 * time.Second
 
 	SamplerIterRefreshPeriod = 5 * time.Minute
+
+	// If set in the context metadata, the requested resource should be written to /
+	// read from this partition ID if the user is authorized to do so.
+	PartitionOverrideKey = "x-buildbuddy-partition-override"
```

Comment:
nit: maybe `x-buildbuddy-cache-partition-override` would be clearer? (could optionally remove `-override` if that's too verbose)